secboot/secboot_sb_test.go

```diff
@@ -2565,3 +2567,119 @@ func (s *secbootSuite) TestSerializedProfile(c *C) {
 		"tpm2-pcr-profile": base64.StdEncoding.EncodeToString([]byte("serialized-profile")),
 	})
 }
+
+type testModel struct {
+	name string
+}
+
+func (tm *testModel) Series() string {
+	return "16"
+}
+func (tm *testModel) BrandID() string {
+	return "some-brand"
+}
+func (tm *testModel) Model() string {
+	return tm.name
+}
+func (tm *testModel) Classic() bool {
+	return false
+}
+func (tm *testModel) Grade() asserts.ModelGrade {
+	return asserts.ModelSecured
+}
+func (tm *testModel) SignKeyID() string {
+	return "some-key"
+}
+
+func (s *secbootSuite) TestResealKeysWithFDESetupHookV1(c *C) {
```

Comment:
What is v1 in this context? Which other formats for keyslots do we have?

Reply:
Actually this is a mistake here. It tests v2. The new format is v3.

We have 2 format of keydata. Keydata are json. v1 is not json. But this does not need resealing.

Reply:
Note, the version here is not related to the key slots. But the key format. Which might be stored in the token for the key slot, or stored in a file.